### PR TITLE
Add spacing between embed error elements

### LINKED ISSUE #3625: Add spacing between embed error elements
**Describe the bug**
When CSS is disabled there is no spacing between the `message` and `link`.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to `http://localhost:7080/mundo/23263889.amp`.
2. Disable CSS.
3. See lack of spacing.

**Expected behaviour**
There should be spacing between the two elements.

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by FK78**
Resolves #3625

**Overall change:** _Add spacing between embed error elements._

**Code changes:**

- _Wrapped the strong element in a `<div>` tag to push link onto new line when no CSS._
- _Update snapshots._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by greenc05** [comment]
Please could you add a screen shot with no CSS @FK78 ?

**Comment by FK78** [comment]
> Please could you add a screen shot with no CSS @FK78 ?

The component would need to be merged into Simorgh first, disabling CSS on Storybook does not work as intended.

Screenshot 2020-07-21 at 16 34 27

**Comment by FK78** [comment]
> Please could you add a screen shot with no CSS @FK78 ?

Screenshot 2020-07-21 at 16 37 26

**Review by rhenshaw56 — APPROVED** [review]

File: packages/components/psammead-embed-error/src/index.jsx [hunk 1]
```diff
@@ -69,7 +69,9 @@ const StyledErrorMessage = styled.div`
 const EmbedError = ({ service, message, fillViewport, link }) => (
   <StyledEmbedError service={service} fillViewport={fillViewport}>
     <StyledErrorMessage service={service}>
-      <strong>{message}</strong>
+      <div>
```
Comment (rhenshaw56) [inline packages/components/psammead-embed-error/src/index.jsx:71]: Little issue, If I move the mouse over some parts of the view port, the link is highlighted, is this expected ?

File: packages/components/psammead-embed-error/src/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -72,9 +72,11 @@ exports[`EmbedError renders a default embed error 1`] = `
...
     class="c1"
   >
-    <strong>
-      Sorry, we can't display this part of the story on this lightweight mobile page.
-    </strong>
+    <div>
+      <strong>
+        Sorry, we can't display this part of the story on this lightweight mobile page.
+      </strong>
+    </div>
```
Comment (simonsinclair) [inline packages/components/psammead-embed-error/src/__snapshots__/index.test.jsx.snap:78]: `div` is a block element, which means we can remove `display: block;` from the `strong` element. 👍

File: packages/components/psammead-embed-error/src/index.jsx (hunk 1 above)
Comment (FK78) [reply packages/components/psammead-embed-error/src/index.jsx:71]: This was present before making these changes.

Comment (rhenshaw56) [reply packages/components/psammead-embed-error/src/index.jsx:71]: you're right, I can see it here https://bbc.github.io/psammead/?path=/story/components-embederror--with-link 😅

**Review by andrewscfc — APPROVED** [review]

**Review by simonsinclair — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
LGTM..

**Pull request merged**